IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/utils/SDKClientUtils.kt

```diff
@@ -179,4 +221,60 @@ internal object SDKClientUtils {
         }
         return true
     }
+
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    @Throws(Exception::class)
+    fun generateSasTokenLatest(resourceUri: String?, key: String?): String? {
+        // Token will expire in one year
+        val expiry = Instant.now().epochSecond + 31536000
```

Comment:
@sw-tt-rajkiranvank @sw-tt-tusharojha take this value in constant 31536000

Reply:
@sw-ot-deveshmevada 
Issue resolved 
Value taken in constant